packages/@cdktf/hcl2cdk/lib/expressions.ts

```diff
@@ -4,8 +4,9 @@ import * as t from "@babel/types";
 import reservedWords from "reserved-words";
 import { camelCase, logger, pascalCase } from "./utils";
 import { TerraformResourceBlock, Scope } from "./types";
-import { getReferencesInExpression } from "@cdktf/hcl2json";
+import { getReferencesInExpression, getExpressionAst } from "@cdktf/hcl2json";
 import { getFullProviderName } from "./provider";
+import * as tfe from "@cdktf/hcl2json/lib/syntax-tree";
```

Comment:
+1 on the top level export

Re: `tfe`, I was going for terraform expression, and I wanted to keep it short, since we use it everywhere. What would you propose?

Reply:
Maybe expression or expr? or individual imports. tfe sounds like terraform enterprise to me :D